rest-server/src/controllers/auth.js

```diff
@@ -44,12 +44,12 @@ const login = (req, res) => {
       }, authConfig.secret, { expiresIn: expiration }, (signError, token) => {
         if (signError) {
           logger.warn('sign token error\n%s', signError.stack);
-          return res.status(500).json({
+          return res.status(401).json({
```

Comment:
INTERNAL SERVER ERROR instead of UNAUTHORIZED

Reply:
ok. But is the error message 'sign token error' accurate to indicate an internal server error?

Reply:
right, it means 'error occurs during server signing token'.

Reply:
sounds good. will change.